examples/with-passport/pages/signup.js

```diff
@@ -19,7 +19,7 @@ const Signup = () => {
       password: e.currentTarget.password.value,
     }
 
-    if (body.password !== e.currentTarget.rpassword.value) {
```

Comment:
`rpassword` here relates to an input with a name set as `rpassword`. The way it was before is correct.